app/lib/service/openid/jwt.dart

```diff
@@ -214,30 +214,26 @@ class JwtPayload extends UnmodifiableMapView<String, dynamic> {
   ///
   /// Returns `false` if the current timestamp is outside of the allowed range.
   /// If the timestamp is missing, we treat it as if it were expired/invalid.
-  bool verifyTimestamps([DateTime? now]) {
+  bool isTimely({DateTime? now, Duration? threshold}) {
     now ??= clock.now();
+    threshold ??= Duration.zero;
 
     bool isABeforeB(String name, DateTime? a, DateTime? b) {
       if (a == null || b == null) {
-        // TODO: remove debug message after the appropriate difference threshold is selected.
-        print('$name is missing.');
+        _logger.info('JWT does not have "$name" field.');
         return false;
       }
       if (a.isBefore(b) || a == b) {
         return true;
       }
```

Comment:
Don't have two kinds of comparisons.

Just do:
```dart
return a.add(threshold).isBefore(b);
```

Reply:
Having multiple comparisons, means that we'll have more code flows... and more options for flipping `<` with `>` and making a mistake. Let's keep it simple.